articles/libraries/lock/v10/index.md

```diff
@@ -114,7 +113,7 @@ function showLoggedIn() {
 This example demonstrates using Lock 10 with a Single Page Application (SPA). To learn how Lock can be modified to provide frictionless authentication for any app, see the [API Reference][lock-api] and the [Configuration Options Reference][lock-customization]. For details specifically about customizing the look and feel of Lock in your app, please take a look at the [UI Customization][ui-customization] page.
 :::
 
-## Start Using Lock
+## Further Examples
```

Comment:
Also this sentence `For more information on using Lock see the documentation.`, that links to itself, should not be here.